great_expectations/expectations/expectation.py

```diff
@@ -1081,6 +1081,9 @@ def validate_metric_value_between_configuration(
 
         try:
             # Ensuring Proper interval has been provided
+            assert (
```

Comment:
I wasn't expecting this to break a bunch of stuff. This seems like a reasonable assertion.